src/scss/_styles.scss

```
@@ -24,6 +24,15 @@ body {
 .container {
   width: 1520px;
   margin: 0 auto;
+
+  @media screen and (max-width: $xxl) {
+    width: 1120px ;
+    margin: 0 auto;
+  }
+
+  @media screen and (max-width: $xl) { //* это я считала размер ширины на 1024 по пропорции, пока это не используется*\\
+    width: 820px;
+  }
```

```suggestion
/* TODO-[YOUR_SURNAME]: [YOUR_COMMENT_ON_ENGLISH] */
/*  @media screen and (max-width: $xl) {  */
/*    width: 820px; */
/*  } */
```
What is not being used is wrapped in a comment